src/vue-components/modals/editElementGeneral.vue

```diff
@@ -130,4 +277,31 @@
     .row, .srow {
         margin-top: 1em;
     }
+
+    .input-button {
```

Comment:
`gridTranslateModal.vue` has a very similar CSS rule for the pronunciation play button. please unify and add it to some common CSS like `custom.css` or even better: extract the input field + play button to an extra component and use it here twice and in translate modal once.